python/mxnet/ndarray/contrib.py

```diff
@@ -548,15 +548,15 @@ def isnan(data):
     """
     return data != data  # pylint: disable=comparison-with-itself
 
-def getRescaleGrad(rescale_grad, ctx=mx.cpu()):
+def _getRescaleGrad(rescale_grad, ctx=mx.cpu()):
```

Comment:
nit pick on the style: we use snake_case for python methods, and underscore prefix for non-public methods.

Reply:
DONE